### PR TITLE
Store release data in custom TPR

### PR DESCRIPTION
**Pull request by tamalsaha**

### CODE REVIEW

**Review by sadlil — APPROVED** [review]

File: client/clientset/fake/release.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/watch"
+)
+
+type FakeRelease struct {
+	Fake *testing.Fake
+	ns   string
+}
+
+var certResource = schema.GroupVersionResource{Group: "appscode.com", Version: "v1beta1", Resource: "releases"}
```
Comment (sadlil) [inline client/clientset/fake/release.go]: This needs renaming.

**Review by sadlil — CHANGES_REQUESTED** [review]

**Review by sadlil — COMMENTED** [review]
If you have plan to merge this PR to original helm, i don't think they will the whole complex client idea we implemented for TPRs. They probably would go with a simple `http` client.

**Comment by tamalsaha** [comment]
What do you mean by simple HTTP client?

**Comment by sadlil** [comment]
I have seen a design in some projects where people uses TPR and uses a `http.Client` with the specified url to access the apis. I was just telling that they may like that approach.

**Comment by tamalsaha** [comment]
Hmm. We will see. I think they will not accept this pr, since it uses TPR which is alpha stuff. https://github.com/kubernetes/helm/issues/1650#issuecomment-280143162

**Comment by TamalSaha** [comment]
@sadlil if they accept, they will not complain about the simple http client. fake is used for testing.

**Pull request merged**